### ISSUE TITLE
NHS service standard: Make your service interoperable

### ISSUE THREAD

**Issue by nancyhalladay24** [nhsuk/nhsuk-service-manual-backlog#360]
Use this issue to discuss point 17 in the NHS service standard: [Make your service interoperable](https://service-manual.nhs.uk/service-standard/17-make-your-service-interoperable)
If you have any comments or experiences that relate to the standards as a whole, please use the GitHub issue for [the NHS service standard](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/336)
## Parent
NHS service standard #336

**Comment by ryma2fhir** [comment]
As a member of the Interoperability Standards Team I have spoken to my manager who would like the following changes to be made. This is to ensure we are aligned to Let's Talk Architecture and the [proposed principles updates](https://future.nhs.uk/Standards_and_Interoperability/messageShowThread?threadID=14937486).

------
The following can be removed. Anything relating to terminology, what data should be recorded or following record standards, e.g. PRSB, should be part of a record keeping section:
•	for disease information in mortality and morbidity statistics, you use [ICD-10 (the World Health Organization's International Classification of Diseases, version 10)](https://icd.who.int/browse10/2016/en)
•	for electronic care records, you use the structured clinical vocabulary [SNOMED CT (NHS Digital)](https://digital.nhs.uk/services/terminology-and-classifications/snomed-ct)
Remove from the NHS Numbers bullet, I don't know what this means, but it doesn't belong here. Looks like more record keeping stuff:
•	and NHS data registers and comply with NHS clinical information standards
Remove or move somewhere else, barcodes are not interop. Also needs a link to scan4safety in its new home:
•	if you use barcodes or similar identifiers, you comply with the GS1 barcodes standard as set out in Scan4Safety
I'd argue that the TCoP line should be in another section since TCoP says almost nothing about interop. However, if it's staying the struck out text needs to be removed since TCoP includes "4. Make use of open standards":
•	you work to open standards and the Technology Code of Practice

This isn't interop, it's making data open when it can be open. There's probably a whole missing section for this in the service standard:
•	if you create any data sets that could be useful to others, you publish them in an open machine-readable format, under an [Open Government Licence](https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/), unless they contain personally identifiable information, sensitive information, or where publishing the data would infringe the intellectual property rights of someone outside the NHS or government
 
Remove since anything they should use should be in the standards directory and I've never once heard this referenced by anybody:
•	where relevant, you comply with the [standards published by the Open Standards Board (GOV.UK)](https://www.gov.uk/government/publications/open-standards-for-government)
 
Remove since we're already telling people to use the standards directory and we don't need to do it twice:
•	you use agreed [FHIR-based APIs (from NHS Digital's API catalogue, filtered by FHIR)](https://digital.nhs.uk/developer/api-catalogue?filter=fhir) to join up care for patients
 
Some of these are too wordy and need simplifying:
•	you have checked use the standards recommended in the [NHS data standards directory](https://data.standards.nhs.uk/)
•	your maximise flexibility and make content, tools and services available to more people through well-designed APIs, following APIs follow NHS [API policies and best practices](https://digital.nhs.uk/developer/guides-and-documentation/api-policies-and-best-practice)
•	where appropriate, you use [NHS number (NHS Digital)](https://digital.nhs.uk/data-and-information/information-standards/information-standards-and-data-collections-including-extractions/publications-and-notifications/standards-and-collections/isb-0149-nhs-number) to identify patients on APIs
 
Missing:
•	you follow this standard for interoperability you build and maintain yourself, and also require that your suppliers follow it
•	you use [FHIR UK Core](https://simplifier.net/hl7fhirukcorer4) when building new APIs or migrating old APIs, for example to add significant new functionality
•	you keep pace with national API and interoperability standards changes
•	your APIs use structured data by default, HTML 5 optionally and PDF never (JT author note, there's a separate line around PDF [elsewhere in the service standard](https://service-manual.nhs.uk/content/pdfs-and-other-non-html-documents), but I want it here specifically also)
•	you use synchronous REST APIs by default
•	if information needs to be presented to a user in a specific format then the system presenting the data is responsible for it, not the system that sent it the data on an API

Thank you,
Ryan

**Comment by sarawilcox** [comment]
Hi @ryma2fhir , thanks for logging this feedback. Would you have time for a chat, please, maybe next week? I have a few questions and things I'd be grateful for clarification on before I kick off work on an update. I'll contact you via Teams.

**Comment by sarawilcox** [comment]
Hi @ryma2fhir, I'm making some initial updates to this standard. I will also need to run things by:
- someone who knows about terminology/clinical coding standards - as discussed
- our clinicians, as we're talking with them about standard 16 on clinical safety
- TaDA and the DHSC assurance team who sign off any changes to the service standard

My initial thoughts are:
- to move the stuff about open standards to [service standard 13](https://service-manual.nhs.uk/standards-and-technology/service-standard-points/13-use-and-contribute-to-open-standards-common-components-and-patterns), which deals with open standards (I'm bringing that standard into closer alignment with GOV's standard 13 anyway)
- to move some stuff over to the clinical safety standard, e.g. the stuff about barcodes 

The mention of NHS data registers is now obsolete.

I can make some changes to shorten the wording where it's overlong, as you suggest. And I'll add the missing points.

I'll then share the content with you again.

**Comment by sarawilcox** [comment]
Spoke with Pete Turnbull who advised to keep SNOMED CT and ICD 10 (and 11 when adopted) in service standard 17. 

Clinical team asked us to NHS number under interoperability.
